### ISSUE TITLE
[BUG] AutoArima fails on exogenous example

### ISSUE THREAD

**Issue by jeremydiba** [carlomazzaferro/scikit-hts#47]
**Describe the bug**
Passing of exogenous variables to hts.model.AutoArimaModel is of incorrect shape and therefore fails.  

**To Reproduce**
Create a Hierarchy tree using training set like seen in examples with exogenous variables:

exogenous = {k: ['SNP', 'HMI'] for k in df_train.columns if k not in ['SNP', 'HMI']}
ht = HierarchyTree.from_nodes(hier, df_train, root='United States', exogenous=exogenous)
hier_df = ht.to_pandas()

Fit using AutoArimaModel

clf = AutoArimaModel(ht, start_p=1, start_q=1, max_p=3, max_q=3, stepwise=True, trace=True, m=12, maxiter=30 , n_jobs=10)
model = clf.fit(df=hier_df, nodes=hier, root='United States')

Create a hierarchy tree using test set 
exo = {k: ['SNP', 'HMI'] for k in df_test.columns if k not in ['SNP', 'HMI']}
ht_test = HierarchyTree.from_nodes(hier, df_test, root='United States', exogenous=exo)

Predict using test hierarchy tree 
predicted_autoarima = model.predict(ht_test.get_node("TX"), steps_ahead=10)

**Expected behavior**
As seen in the class hts/model/ar.py predict(), if exogenous is set to true, it sets exogenous to node.item.   In this case, node.item contains the exogenous variables as well as the node itself.  this makes the size of the dataframe size(exogenous) + 1
This is confirmed by the error message 
Provided exogenous values are not of the appropriate shape. Required (10, 2), got (10, 3).  
The solution as I see it would be setting exogenous to the proper slice (node.item.iloc[:,1:])

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - scikit-hts version: 0.5.3
- Python version: 3.8.3

**Additional context**
Reassigning the node to 
test_node = ht_test.get_node("TX")
test_node.item = test_node.item.iloc[:,1:] and passing that into predict is not a workaround
